pkg/queue/manager.go

```diff
@@ -306,6 +308,20 @@ func (m *Manager) UpdateLocalQueue(q *kueue.LocalQueue) error {
 	return nil
 }
 
+func (m *Manager) GetLQSnapshot(qkey string) *LocalQueue {
+	m.Lock()
+	defer m.Unlock()
+	qImpl, ok := m.localQueues[qkey]
+	if !ok {
+		return nil
+	}
+	return &LocalQueue{
+		Key:          strings.Clone(qImpl.Key),
+		Labels:       maps.Clone(qImpl.Labels),
+		ClusterQueue: qImpl.ClusterQueue,
+	}
+}
```

Comment:
I still think that we should take lq labels from cache. Basically, taking queue lock affect on a whole of scheduling performance. For this feature, we do not want to incrase the scheduling cycle.